tests/common/docker_image.py

```diff
@@ -337,6 +337,90 @@ def wait(self, max_attempts: int = 15, sleep_rate: float = 1.0) -> bool:
         return False
 
 
+class SOEFDockerImage(DockerImage):
+    """Wrapper to SOEF Docker image."""
+
+    PORT = 12002
```

Comment:
I think we don't need anymore if we want it to be anything other than 9002 (the original value). If it works with any other port, we can just have the parameterised self._port assigned to all parts (marked below).

Reply:
TBH there's a benefit from defining it here for clarity. There's no cost to keeping it imo.